### PR TITLE
Feat: keypad dtmf sound

### PR DESCRIPTION
**Pull request by digiboridev**

### CODE REVIEW

File: lib/features/keypad/widgets/keypad.dart [hunk 1]
```diff
@@ -29,7 +31,10 @@ class Keypad extends StatelessWidget {
           KeypadKeyButton(
             text: k.text,
             subtext: k.subtext,
-            onKeyPressed: (v) => onKeypadPressed(v),
+            onKeyPressed: (v) {
+              Dtmf.playTone(digits: v, samplingRate: 80000.0);
```
Comment (ycherniavskyi) [inline lib/features/keypad/widgets/keypad.dart:35]: Do we really need such big `sampling_Rate`? What if remove this property and use default value that is currently `500`? Will it some howe affect the audible DTMF tone quality?

File: lib/features/settings/features/about/view/about_screen_style.dart [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
```
Comment (ycherniavskyi) [inline lib/features/settings/features/about/view/about_screen_style.dart:2]: Do not add changed not related to actual PR functionality. Use separate PR or just `refactor` commit.

File: lib/theme/extension/logo_assets.dart [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/material.dart';
```
Comment (ycherniavskyi) [inline lib/theme/extension/logo_assets.dart:2]: Do not add changed not related to actual PR functionality. Use separate PR or just `refactor` commit.

File: packages/_web_socket_channel/pubspec.yaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 name: _web_socket_channel
 description: Internal package to create platform-appropriate WS clients..
 version: 1.0.0
+publish_to: none
```
Comment (ycherniavskyi) [inline packages/_web_socket_channel/pubspec.yaml:4]: Do not add changed not related to actual PR functionality. Use separate PR or just `refactor` commit.

**Review by ycherniavskyi — CHANGES_REQUESTED** [review]
It will be great to recreate this PR from `main` branch, which then will be merged to `beta/develop`.

File: lib/features/settings/features/about/view/about_screen_style.dart (hunk 2 above)
Comment (digiboridev) [reply lib/features/settings/features/about/view/about_screen_style.dart:2]: Its already from PR https://github.com/WebTrit/webtrit_phone/pull/159

File: lib/theme/extension/logo_assets.dart (hunk 3 above)
Comment (digiboridev) [reply lib/theme/extension/logo_assets.dart:2]: Its already from PR https://github.com/WebTrit/webtrit_phone/pull/159

File: packages/_web_socket_channel/pubspec.yaml (hunk 4 above)
Comment (digiboridev) [reply packages/_web_socket_channel/pubspec.yaml:4]: Its already from PR https://github.com/WebTrit/webtrit_phone/pull/159

File: lib/features/keypad/widgets/keypad.dart (hunk 1 above)
Comment (digiboridev) [reply lib/features/keypad/widgets/keypad.dart:35]: Its ios internal kitchen, using 80000 sounds like default tones.

**Review by ycherniavskyi — APPROVED** [review]

**Pull request merged**